src/main/scala/org/ergoplatform/modifiers/history/header/Header.scala

```diff
@@ -149,7 +149,7 @@ object Header extends ApiCodecs {
       "powSolutions" -> h.powSolution.asJson,
       "nBits" -> h.nBits.asJson,
       "height" -> h.height.asJson,
-      "difficulty" -> h.requiredDifficulty.toString.asJson,
+      "difficulty" -> h.requiredDifficulty.toLong.asJson,
```

Comment:
Currently, it is hard to say which services may use this field. So better to update openapi.yaml. not the actual implementation.